smoke-tests/framework/build.gradle.kts

```diff
@@ -2,6 +2,12 @@ plugins {
   id("ai.java-conventions")
 }
 
+tasks {
+  named("spotbugsMain") {
+    enabled = false
```

Comment:
should we enable it by default?

Reply:
additionally, should the task log for spotbugsMain shows up in `Build / assemble / assemble (pull_request)`?

Reply:
> should we enable it by default?

it's only disbled here for `:smoke-tests:framework` module since that's just test code

> additionally, should the task log for spotbugsMain shows up in `Build / assemble / assemble (pull_request)`?

it shows up (and fails) the tests builds (which run the `gradle check`)